### PR TITLE
Add TabPage tests and cleanup code

### PR DESCRIPTION
**Pull request by hughbe**
###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/winforms/pull/1482)

### CODE REVIEW

File: src/System.Windows.Forms/src/System/Windows/Forms/Control.cs [hunk 1]
```diff
@@ -8972,8 +8972,7 @@ internal virtual void OnInvokedSetScrollPosition(object sender, EventArgs e)
         [EditorBrowsable(EditorBrowsableState.Advanced)]
         protected virtual void OnLocationChanged(EventArgs e)
         {
-            OnMove(EventArgs.Empty);
-
+            OnMove(e);
```
Comment (zsd4yr) [inline src/System.Windows.Forms/src/System/Windows/Forms/Control.cs]: bug fix?

Comment (hughbe) [reply src/System.Windows.Forms/src/System/Windows/Forms/Control.cs]: Essentially - see the 3rd commit :)

File: src/System.Windows.Forms/src/System/Windows/Forms/TabPage.cs [hunk 2]
```diff
@@ -35,84 +32,79 @@ public class TabPage : Panel
         private bool useVisualStyleBackColor = false;
 
         /// <summary>
-        ///  Constructs an empty TabPage.
+        /// Constructs an empty TabPage.
```
Comment (RussKie) [inline src/System.Windows.Forms/src/System/Windows/Forms/TabPage.cs]: This is not correct spacing for the xml-doc
Please see https://github.com/dotnet/winforms/issues/1489 guideline 2.

Likely we'll need to do a sweep across the codebase to have it fixed.

Comment (hughbe) [reply src/System.Windows.Forms/src/System/Windows/Forms/TabPage.cs]: Fixed

**Pull request merged**